core/src/main/resources/hudson/model/Job/buildTimeTrend_resources.js

```diff
@@ -67,76 +67,11 @@ window.buildTimeTrend_displayBuilds = function (data) {
 /**
  * Generate SVG Icon
  */
-function generateSVGIcon(iconName, iconSizeClass) {
-  const imagesURL = document.head.getAttribute("data-imagesurl");
-
-  const isInProgress = iconName.endsWith("anime");
-  let buildStatus = "never-built";
-  switch (iconName) {
-    case "red":
-    case "red-anime":
-      buildStatus = "last-failed";
-      break;
-    case "yellow":
-    case "yellow-anime":
-      buildStatus = "last-unstable";
-      break;
-    case "blue":
-    case "blue-anime":
-      buildStatus = "last-successful";
-      break;
-    case "grey":
-    case "grey-anime":
-    case "disabled":
-    case "disabled-anime":
-      buildStatus = "last-disabled";
-      break;
-    case "aborted":
-    case "aborted-anime":
-      buildStatus = "last-aborted";
-      break;
-    case "nobuilt":
-    case "nobuilt-anime":
-      buildStatus = "never-built";
-      break;
-  }
+function generateSVGIcon(iconName) {
+  const icons = document.querySelector("#jenkins-build-status-icons");
+  iconName = iconName.replace("-anime", "");
```

Comment:
any reason why the anime svgs are excluded? Before they were animated when the build was running

Reply:
This is a fun unaddressed comment to find after triaging https://github.com/jenkinsci/jenkins/pull/8705#issuecomment-1943677731